.github/ISSUE_TEMPLATE/1.bug.yml

```diff
@@ -8,49 +8,52 @@ body:
         Please provide a searchable summary of the issue in the title above ⬆️.
 
         Thanks for contributing by creating an issue! ❤️
-  - type: checkboxes
+
+  - type: input
     attributes:
-      label: Duplicates
-      description: Please [search the history](https://github.com/mui/mui-x/issues) to see if an issue already exists for the same problem.
-      options:
-        - label: I have searched the existing issues
-          required: true
+      label: Search keywords
+      description: Your issue may have already been reported! List the keywords you've used to search the [existing issues](https://github.com/mui/mui-x/issues). This will also make your issue searchable for others.
+      placeholder: e.g. datagrid column resizing
+    validations:
+      required: true
   - type: checkboxes
     attributes:
       label: Latest version
       description: We roll bug fixes, performance enhancements, and other improvements into new releases.
       options:
         - label: I have tested the latest version
           required: true
+
   - type: textarea
     attributes:
-      label: Steps to reproduce 🕹
+      label: Steps to reproduce
       description: |
-        **⚠️ Issues that we can't reproduce will be closed.**
-
-        Please provide a link to a live example and an unambiguous set of steps to reproduce this bug. As a starting point, we recommend you [browse the documentation](https://mui.com/x/introduction/), and [select](https://mui.com/static/docs/forking-an-example.png) the closest example to your use case. Or you can use a [basic template](https://mui.com/r/x-issue-template) to build your reproduction case.
+        **⚠️ Please provide a live example for your report⚠️**
+        If you don't have one, you can use one of these options:
+          - [DataGrid codesandbox template](https://codesandbox.io/s/github/mui/mui-x/tree/master/templates/x-data-grid?file=/demo.tsx)
+          - Fork any of the examples in our [documentation](https://mui.com/x/introduction/) by [clicking on the codesandbox or stackblitz icon](https://mui.com/static/docs/forking-an-example.png)
       value: |
-        Link to live example:
+        Link to live example: (required)
 
         Steps:
         1.
         2.
         3.
   - type: textarea
     attributes:
-      label: Current behavior 😯
+      label: Current behavior
       description: Describe what happens instead of the expected behavior.
   - type: textarea
     attributes:
-      label: Expected behavior 🤔
+      label: Expected behavior
       description: Describe what should happen.
   - type: textarea
     attributes:
-      label: Context 🔦
+      label: Context
       description: What are you trying to accomplish? How has this issue affected you? Providing context helps us come up with a solution that is more useful in the real world.
   - type: textarea
     attributes:
-      label: Your environment 🌎
+      label: Your environment
```

Comment:
The icons are colored and have a big visual weight, they catch the eye a bit more than I'd like them to. In bug reports particularly, being able to focus on details is important. I've left them for questions/feature-requests because they're less annoying there.

Reply:
The arguments make sense, maybe there is a bit too much color variance in the chosen icons.
But I have one nitpick regarding this: now it would be harder to skimp through the issue and immediately notice, where a given section ends and a new one begins because users can create their own headings or bolded sections and that could potentially make the issue parsing/glancing a bit harder.
But I'm all for experimenting and adjusting if it is actually a problem. 👌